portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+public final class Comment {
+  
+  public static final String entityNameParam = "comment";
```

Comment:
nit: The Google Java style guide requires that you use capital letters and underscore separators for the names of constants, like:
```
public static final String ENTITY_NAME_PARAM = ...
```

Reply:
Done.